api/friendsLists.js

```diff
@@ -1 +1,98 @@
-//friendsLists file - delete this if you're writing this file
\ No newline at end of file
+const express = require("express");
+const friendsRouter = express.Router();
+
+const {
+  addFriends,
+  getUserByUsername,
+  removeFriend,
+  getFriendsByUserId,
+  getPostsByUserId,
+} = require("../db");
+
+const { requireUser } = require("./utils");
+
+friendsRouter.post(
+  "/friends/:friendId",
```

Comment:
1. At this point we are at "/api/friendslists/friends/:friendId" so we should remove "/friends" from this but I do also like the idea of making this file's route "api/friends" - less verbose and still clearly defines what it is
2. You are using params but trying to get a username when only an ID is in the params. So we either have to extract the ID or we have to change the endpoint to "/:friendUsername"